packages/fxa-auth-server/lib/senders/sms.js

```diff
@@ -7,7 +7,6 @@
 const Cloudwatch = require('aws-sdk/clients/cloudwatch');
 const error = require('../error');
 const MockSns = require('../../test/mock-sns');
-const P = require('bluebird');
```

Comment:
Converting this file / tests was interesting. The `setImmediate`s used in the tests, which were there to allow the `pollCurrentSpend` to run, aren't a sound way to ensure that it completed, and was leading to assertion races. I believe bluebird is more predictable in how it interacts with the event loop than native promises, but we shouldn't be relying on that. I refactored in order to manually trigger and wait for that function to complete. I then went on to convert to `await` and added a `close` function to stop the polling.